keywords.txt

```diff
@@ -33,6 +33,7 @@ NtpClock	KEYWORD1
 SystemClock	KEYWORD1
 SystemClockLoop	KEYWORD1
 SystemClockCoroutine	KEYWORD1
+StmRTCClock	KEYWORD1
```

Comment:
nit: I think I would prefer `StmRtcClock`, because it's more consistent with "NtpClock", and the capitalized "RTCC" part run together and is not as readable